qa-pipelines/tasks/lib/cf-deploy-upgrade-common.sh

```diff
@@ -165,6 +175,15 @@ set_uaa_sizing_params() {
 }
 
 set_scf_sizing_params() {
+    if [[ ${cap_platform} == "eks" ]] ; then
+        HELM_PARAMS+=(--set sizing.cc_uploader.capabilities={"SYS_RESOURCE"})
```

Comment:
All these lines can be combined using a brace expansion

Reply:
yeah, but it gives no advantage. I find this more readable

Reply:
Avoiding writing the same thing multiple times is almost always preferable. Same reason we're templating the qa-pipeline.yml file; readability is subjective (and I'd argue the compact form using brace expansion is still more readable), repeating the same thing multiple times introduces more opportunity for error, and makes errors harder to catch. Less unnecessary, repetitive code, and less room for error *is* the advantage.